vital/algo/algorithm.h

```diff
@@ -173,49 +157,43 @@ class VITAL_ALGO_EXPORT algorithm
 protected:
   algorithm(); // CTOR
 
-  /**
-   * \brief Attach logger to this object.
-   *
-   * This method attaches a logger to this object. The name supplied
-   * is used to name the logger. Since this is a fundamental base
-   * class, derived classes will want to have the logger named
-   * something relevant to the concrete algorithm.
-   *
-   * A logger is attached by the base class, but it is expected that
-   * one of the derived classes will attach a more meaningful logger.
-   *
-   * \param name Name of the logger to attach.
-   */
+  /// \brief Attach logger to this object.
+  ///
+  /// This method attaches a logger to this object. The name supplied
+  /// is used to name the logger. Since this is a fundamental base
+  /// class, derived classes will want to have the logger named
+  /// something relevant to the concrete algorithm.
+  ///
+  /// A logger is attached by the base class, but it is expected that
+  /// one of the derived classes will attach a more meaningful logger.
+  ///
+  /// \param name Name of the logger to attach.
   void attach_logger( std::string const& name );
 
 private:
-  /**
-   * \brief Logger handle.
-   *
-   * This handle supplies a logger for all derived classes.
-   */
+  /// \brief Logger handle.
+  ///
+  /// This handle supplies a logger for all derived classes.
   kwiver::vital::logger_handle_t m_logger;
 
   std::string m_impl_name;
 };
 
 // ------------------------------------------------------------------
 /// An intermediate templated base class for algorithm definition
-
-/**
- *  Uses the curiously recurring template pattern (CRTP) to declare the
- *  clone function and automatically provide functions to register
- *  algorithm, and create new instance by name.
- *  Each algorithm definition should be declared as shown below
- *  \code
-    class my_algo_def
-    : public algorithm_def<my_algo_def>
-    {
-      ...
-    };
-    \endcode
- *  \sa algorithm_impl
- */
+///
+///  Uses the curiously recurring template pattern (CRTP) to declare the
+///  clone function and automatically provide functions to register
+///  algorithm, and create new instance by name.
+///  Each algorithm definition should be declared as shown below
+///  \code
+///  class my_algo_def
+///  : public algorithm_def<my_algo_def>
+///  {
+///    ...
+///  };
+///  \endcode
+///  \sa algorithm_impl
```

Comment:
So... this in in the original, but just noting there's an extra space after the `///` here.